src/pf-tooltip/index.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+<html>
+<head>
+    <title>pf-tooltip example</title>
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.13.0/css/patternfly-additions.css">
+    <link rel="stylesheet" href="/dist/css/patternfly.css">
+    <script src="//cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/0.7.22/webcomponents.js"></script>
+    <script src="/dist/js/pf-utils.js"></script>
+    <script src="/dist/js/pf-tooltip.js"></script>
+</head>
+<body>
+<div class="container">
+    <div class="page-header">
+        <h1>Tooltips</h1>
+    </div>
+    <hr>
+
+    <pf-tooltip placement="left" target="#btn-left">
+        Tooltip text on Left
+    </pf-tooltip>
+    <button id="btn-left" type="button" class="btn btn-primary">Left</button>
+
+    <pf-tooltip placement="top" target="#btn-top">
+        <span>Tooltip on top</span>
+    </pf-tooltip>
+    <button id="btn-top" type="button" class="btn btn-secondary">Top</button>
+
+    <pf-tooltip placement="bottom" target="#btn-bottom">
+        <span>Tooltip on bottom</span>
+    </pf-tooltip>
+    <button id="btn-bottom" type="button" class="btn btn-warning">Bottom</button>
+
+    <pf-tooltip placement="right" target="#btn-right">
+        <span>Tooltip on right</span>
+    </pf-tooltip>
+    <button id="btn-right" type="button" class="btn btn-danger">Right</button>
+
+    <br/>
+    <br/>
+    <script>
+        setTimeout(function(){
+          var pfTooltipRight = document.querySelector('[target="#btn-right"]');
+          pfTooltipRight.innerHTML = "Right content changed!";
+
+          var event = new CustomEvent('handleContentChanged',{});
+          pfTooltipRight.dispatchEvent(event);
```

Comment:
I'd rather avoid another plugin for this sort of thing if possible. The main reason I added this was to showcase the ability to re-render content after the tooltip content changes. This will be useful from the downstream framework perspective, as you might decide to render the content with a binding and then subsequently change that binding's value. Ideally, an Angular or React or Framework X shim handles this completely. I can write up an example of this next...

In React it might look like this:
```
<Tooltip placement="left" targetSelector="#btn-left"> {this.state.tooltipContent} <Tooltip>
```

where `Tooltip` is a React Component which renders a `<pf-tooltip>` internally and then fires this event when its bound value changes. Transclusion here in the React component is simply just rendering with {this.props.children} as the passed in `this.state.tooltipContent`. It will automatically fire a React lifecycle event, like `componentDidUpdate` when this content changes (when in turn could fire the web components `handleContentChanged`) .  

Hope this makes sense...it should allow for a seamless integration in the React ecosystem that behaves exactly like other React components (only the React shim is disguising this and rendering a web component for shared encapsulation of view logic).

I would think something in Angular directive would be very similar...